rust/publish-ghost/src/lib.rs

```diff
@@ -76,6 +77,26 @@ pub struct Cli {
     #[arg(long, conflicts_with = "push")]
     pull: bool,
 
+    // Push as draft
```

Comment:
This and other args should have three slashes so that they appear in help text

```suggestion
    /// Push as draft
```